samza-api/src/main/java/org/apache/samza/system/descriptors/GenericSystemDescriptor.java

```diff
@@ -16,12 +16,9 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.samza.operators.descriptors;
+package org.apache.samza.system.descriptors;
```

Comment:
Yeah, I had separate 'systems.' and 'streams.' descriptor packages earlier. However there's not corresponding 'streams' package in rest of Samza. Also, putting system and stream descriptors in separate packages introduces method visibility issues. E.g., it requires making GenericInputDescriptor constructor public, although it shouldn't be since they should always be obtained using GenericSystemDescriptor. So I thought it simpler to put them all in the existing 'system' package. In documentation and SEP I've been referring to them as "System and Stream Descriptors", so it shouldn't be that confusing.